src/lang/lwc_cn.properties

```diff
@@ -22,23 +22,31 @@ you=你
 chest=箱子
 furnace=熔炉
 dispenser=发射器
+dropper=Dropper
```

Comment:
Please don't add keys if you are not going to translate them. Untranslated keys default to the English locale anyways. If you do this, the translation will need to be changed in ALL locale files every time an English translation changes. This is very inefficient/tedious.